src/NuGet.Frameworks/NuGetFramework.cs

```diff
@@ -150,11 +150,11 @@ public string DotNetFrameworkName
                         parts.Add(String.Format(CultureInfo.InvariantCulture, "Profile={0}", Profile));
                     }
 
-                    result = String.Join(", ", parts);
+                    result = String.Join(",", parts);
```

Comment:
Removing the space is fine. Just make sure all unit tests are updated to match the change.